### ISSUE TITLE
Strutta responsabile del servizio non presente nella scheda Servizi

### ISSUE THREAD

**Issue by danilomariotti** [pcrt/joomla-italia-theme#33]
Steps to reproduce the issue

- Caricare un servizio 
- Visualizzare il servizio

Expected result

- Durante il caricamento dovrebbero essere disponibili i campi per caricare la struttura responsabile
- Nella visualizzazione del servizio dovrebbe apparire la struttura responsabile 

Actual result

- Durante il caricamento non sono disponibili campi per caricare la struttura responsabile, ho ricontrollato la documentazione ma non mi sembra di avere trovato nulla a riguardo
- Nella visualizzazione del servzio non appare (ovviamene) la struttura responsabile. Questo genera anche un errore nell'app di valutazione del sito

**Comment by pcrt** [comment]
Dovresti creare un nuovo custom field di tipo sql come descritto nella documentazione.
https://github.com/pcrt/joomla-italia-theme/blob/main/doc/custom-field-articoli.md#servizi-di-cui-la-struttura-%C3%A8-responsabile
Seguendo questa procedura dovresti essere in grado di selezionare gli articoli delle strutture, quindi anche passare la validazione.

**Comment by danilomariotti** [comment]
La procedura riportata a quel link serve per poter scegliere l'elenco dei servizi presenti nella struttura (nella scheda strutture). A me serve poter scegliere l'elenco delle strutture responsabili del servizio (nella scheda servizi). Forse ho interpretato male?

**Comment by danilomariotti** [comment]
@pcrt intendevi che devo scrivere un'istruzione sql simile a quella indicata cambiando l'id finale della categoria, in modo da avere l'elenco delle strutture?

**Comment by Knock-Out** [comment]
@danilomariotti Si.

![Articoli  Modifica campo aggiuntivo - Joomla Italia Theme - Amministrazione](https://github.com/pcrt/joomla-italia-theme/assets/24791536/7d64e022-8b51-4ff3-96b4-d956ea1653f9)

![Articoli  Modifica - Joomla Italia Theme - Amministrazione](https://github.com/pcrt/joomla-italia-theme/assets/24791536/0696a187-3981-4b06-b05d-ebbd9860006c)

![Joomla Italia Theme - Scuola primaria](https://github.com/pcrt/joomla-italia-theme/assets/24791536/4fde6fdb-df57-4332-9c65-374817197604)

**Comment by danilomariotti** [comment]
Ok, fatto. C'è riscontro anche nell'app valutazione. 
Occorre integrare la documentazione però, se non sbaglio.

**Comment by Knock-Out** [comment]
No, è [descritta. ](https://github.com/pcrt/joomla-italia-theme/blob/main/doc/custom-field-articoli.md#servizi-di-cui-la-struttura-%C3%A8-responsabile)
